data/json/items/gun/40mm.json

```
@@ -180,5 +180,28 @@
     "clip_size": 3,
     "modes": [ [ "DEFAULT", "single", 1, "NPC_AVOID" ], [ "MULTI", "multi", 3, [ "NPC_AVOID", "SIMULTANEOUS" ] ] ],
     "proportional": { "weight": 1.5, "volume": 1.8, "price": 2 }
+  },
+  {
+    "id": "pseudo_m203",
```

```suggestion
    "id": "m203_array",
```
This isn't really a pseudo item.

---

Isn't it? It's just a bunch of m203's strapped onto the bot, not an actual gun.